Library/Formula/stack.rb

```diff
@@ -0,0 +1,10 @@
+class Stack < Formula
+  desc "The Haskell Tool Stack"
+  homepage "https://www.stackage.org/"
+  url "https://github.com/commercialhaskell/stack/releases/download/v0.1.2.0/stack-0.1.2.0-x86_64-osx.gz"
```

Comment:
We keep some versioned formulae in [Homebrew/versions](https://github.com/Homebrew/homebrew-versions) but unfortunately GHC isn’t here, so we can’t use an older version.